src/components/modals/NightOrderModal.vue

```diff
@@ -165,7 +165,7 @@ export default {
 }
 
 h3 {
-  margin: 0 40px;
+  margin: 0 90px;
```

Comment:
This adds an extra black gap on the sides of the night order modal - I think it looked better before.

Reply:
Without this it isn't possible to click on the maximize icon in some cases - because the title overlays it.